merge/set_test.go

```diff
@@ -124,11 +124,11 @@ func TestUpdateSet(t *testing.T) {
 			Object: `
 				list:
 				- a
-				- aprime
 				- b
+				- aprime
```

Comment:
I've very carefully inspected these two tests and the change is correct. Ordering isn't broken.